official/resnet/resnet.py

```diff
@@ -516,6 +572,10 @@ def __init__(self, resnet_size_choices=None):
         '--data_dir', type=str, default='/tmp/resnet_data',
         help='The directory where the input data is stored.')
 
+    self.add_argument(
+        '--parallel_calls', type=int, default=5,
+        help='The number of input records to process in parallel.')
```

Comment:
Perhaps mention that this should be equal to the number of CPU cores here as well.